product_docs/docs/biganimal/release/administering_cluster/04_customizing_policy_ definitions.mdx

```diff
@@ -0,0 +1,208 @@
+
+- **Virtual networks should be protected by Azure Firewall**
+
+ BigAnimal does not enable the Azure Firewall. Instead, BigAnimal uses Azure Network Security Group whitelists to specify permissible inbound and outbound traffic.
+
+ If your organization requires an Azure Firewall for compliance purposes, contact [support@biganimal.com](mailto:support@biganimal.com).
+
+#### Manage access and permissions
+
+- **Privileged containers should be avoided**
+
+ BigAnimal must run some containers in the privileged mode for control plane functionality, such as securing and monitoring the application.
+
+- **Immutable (read-only) root filesystem should be enforced for containers**
+
+ Some BigAnimal containers must run with a read-only root filesystem for control plane functionality. This capability is necessary, for example, to use system calls to secure and monitor the BigAnimal application.
+
+- **Running containers as root user should be avoided**
+
+ BigAnimal must run some containers as the root user to provide some aspects of control plane functionality, such as logging. BigAnimal tightly restricts the use of the root user, and no containers running as root expose network connectivity.
+
+- **Containers sharing sensitive host namespaces should be avoided**
+
+ BigAnimal must run some containers that can share the host process ID namespace to monitor network traffic statistics for cluster worker nodes. To prevent traffic sniffing and configuration changes to the worker node system, BigAnimal has removed *all* security capabilities for those containers.
+
+- **Container with privilege escalation should be avoided**
+
+ To enable some monitoring capabilities for Kubernetes, BigAnimal must run some containers that may allow privilege escalation. 
+
+#### Implement security best practices
+
+- **Kubernetes clusters should disable automounting API credentials**
+
+ Microsoft recommends disabling automounting API credentials to prevent a potentially compromised pod from running API commands against a Kubernetes cluster. 
+
+ BigAnimal creates service accounts and roles with the least privileges for Kubernetes operators and operands to prevent this scenario.
+
+#### Enable auditing and logging
+
+Microsoft recommends enabling diagnostic logs in Kubernetes services, Key Vault, and Virtual Machine Scale Sets.
+
+BigAnimal does not enable diagnostic logs for Kubernetes services and Key Vault, but it does enable diagnostic logs for Virtual Machine Scale Sets. Resources managed by BigAnimal are logged in Virtual Machine Scale Sets logs. If you must enable other logs for compliance purposes, contact [support@biganimal.com](mailto:support@biganimal.com).
+
+#### Enable enhanced security features
+
+Microsoft Defender for Cloud now includes the capabilities of Microsoft Defender for open-source relational databases. 
+
+BigAnimal does not enable any of the following capabilities:
+- Microsoft Defender for servers
```

Comment:
Should "servers" be capitalized?

Reply:
It isn't capitalised in the MSDN docs here. https://docs.microsoft.com/en-us/azure/defender-for-cloud/defender-for-servers-introduction.